OmiseSwiftTests/Fixtures/api.omise.co/links/link_test_565dgexs3bvcsjyvmpm-get.json

```diff
@@ -1,25 +0,0 @@
-{
-  "object": "link",
-  "id": "link_test_565dgexs3bvcsjyvmpm",
-  "livemode": false,
-  "location": "/links/link_test_565dgexs3bvcsjyvmpm",
-  "amount": 1490000,
-  "currency": "thb",
-  "used": false,
-  "multiple": false,
-  "title": "B&W P7 Wireless",
-  "description": "Buying Bowers & Wilkins P7 wireless from us",
```

Comment:
🎧 😆

Reply:
I have this charge for a long long time :P